src/Raven.Server/Documents/Handlers/Debugging/DebugPackage/Analyzers/Database/TasksInfoAnalyzer.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Raven.Client.Documents.Operations.Backups;
+using Raven.Client.Documents.Operations.ETL.Queue;
+using Raven.Client.Documents.Operations.OngoingTasks;
+using Raven.Client.Documents.Subscriptions;
+using Raven.Client.ServerWide.Operations;
+using Raven.Server.Dashboard;
+using Raven.Server.Documents.ETL.Handlers;
+using Raven.Server.Documents.ETL.Stats;
+using Raven.Server.Documents.Handlers.Debugging.DebugPackage.Analyzers.Errors;
+using Raven.Server.Documents.Handlers.Debugging.DebugPackage.Analyzers.Issues;
+using Raven.Server.Documents.Handlers.Debugging.DebugPackage.Analyzers.Results.Database;
+using Raven.Server.Web.System;
+
+namespace Raven.Server.Documents.Handlers.Debugging.DebugPackage.Analyzers.Database;
+
+public class TasksInfoAnalyzer(
+    string databaseName,
+    DebugPackageAnalyzeErrors errors,
+    DebugPackageAnalysisIssues issues) : AbstractDebugPackageDatabaseAnalyzer(databaseName, errors, issues)
+{
+    private DateTime? _debugInfoCollectedAt;
+    public TasksAnalysisInfo TasksInfo { get; set; }
+
+    protected override bool RetrieveAnalyzerInfo(DebugPackageEntries entries)
+    {
+        if (entries.TryGetEntry<OngoingTasksHandler>(x => x.GetOngoingTasks(), out var ongoingTasksEntry) == false)
+        {
+            AddWarning("Failed to get ongoing tasks");
+        }
+
+        if (entries.TryGetValue<OngoingTasksHandler, OngoingTasksResult>(x => x.GetOngoingTasks(), out var ongoingTasks) == false)
+        {
+            AddWarning("Failed to get ongoing tasks");
+        }
+
+        if (entries.TryGetValue<SubscriptionsHandler, List<SubscriptionState>>(x => x.GetAll(), "Results", out var subscriptionStates) == false)
+        {
+            AddWarning("Failed to get subscriptions");
+        }
+
+        if (entries.TryGetValue<EtlHandler, List<EtlTaskStats>>(x => x.Stats(), "Results", out var etlStats) == false)
+        {
+            AddWarning("Failed to get ETL task stats");
+        }
+
+        if (entries.TryGetValue<EtlHandler, List<EtlTaskProgress>>(x => x.Progress(), "Results", out var etlProgress) == false)
+        {
+            AddWarning("Failed to get progress of ETL tasks");
+        }
+
+        if (ongoingTasksEntry.TryGetJson("@metadata", out JsonElement metadata) && metadata.TryGetProperty("DateTime", out var dateTimeField) &&
+            dateTimeField.TryGetDateTime(out var retrievalInfoDateTime))
+        {
+            _debugInfoCollectedAt = retrievalInfoDateTime;
+        }
+
+        var taskCounts = new DatabaseOngoingTasksInfoItem();
+
+        TasksInfo = new TasksAnalysisInfo { TaskCounts = taskCounts };
+
+        foreach (var ongoingTask in ongoingTasks.OngoingTasks)
+        {
+            taskCounts.Total++;
+
+            switch (ongoingTask.TaskType)
```

Comment:
For 7.x - we need the AI features here as well, no?

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-24757/Debug-package-analyzer-Handle-AI-tasks-in-TasksInfoAnalyzer-in-7.x